tests/integration/plotter_test.py

```diff
@@ -0,0 +1,24 @@
+import requests as r
+from decouple import config
+
+
+def test_call_endpoint():
+    response = r.get(url=config("PLOTTER_ENDPOINT"))
+    assert response.status_code == 200
+
+
+def test_plot_2d():
+    file_path = "./data/test.csv"
+
+    files = {("file", open(file_path, "rb"))}
+    payload = {
```

Comment:
For now it's okay but remind that we will later deal with very complex models. And we won't need to specify all parameters, so this is fine.

Reply:
Will be done on next PR.